conference.js

```diff
@@ -2305,7 +2305,7 @@ export default {
         eventEmitter.emit(JitsiMeetConferenceEvents.BEFORE_HANGUP);
 
         let requestFeedbackPromise = requestFeedback
-                ? APP.UI.requestFeedbackOnHangup()
+                ? APP.UI.requestFeedbackOnHangup(room)
```

Comment:
@saghul, this is where the conference is left and feedback is requested at the same time. This feedback is essentially post-call feedback but sending of feedback relies on the conference (stored in 'room') to be available. I ended up passing it into the feedback dialog so I have a reference even after the conference has been left.